packages/editor-ui/src/plugins/i18n/locales/en.json

```
@@ -1095,5 +1094,97 @@
 			"missingWorkflow": "Missing workflow",
 			"arrayEmpty": "Must have an array of nodes"
 		}
-	}
+	},
+	"404_NOT_FOUND_MESSAGE": "Oops, couldn’t find that",
+	"BASIC_INFORMATION": "Basic Information",
+	"CHANGE_PASSWORD": "Change Password",
+	"CHECK_INBOX_AND_SPAM": "Please check your inbox (and perhaps your spam folder)",
+	"CONFIRM_DATA_HANDLING_AFTER_DELETION": "What should we do with their data?",
+	"CONFIRM_USER_DELETION": "Are you sure you want to delete this invited user?",
+	"CURRENT_PASSWORD": "Current password",
+	"DELETE": "Delete",
+	"DELETE_CONFIRMATION_MESSAGE": "Type “delete all data” to confirm",
+	"DELETE_CONFIRMATION_TEXT": "delete all data",
+	"DELETE_USER": "Delete {user}",
+	"DELETE_WORKFLOWS_CREDENTIALS": "Delete their workflows and credentials",
+	"EMAIL": "Email",
+	"EMAIL_INVITES_SENT": "An invite email was sent to {emails}",
```

```suggestion
	"EMAIL_INVITES_SENT_TITLE": "An invite email was sent to {emails}",
```

Same for others where this happens.